### PR TITLE
release 4.27.0 changelog

### PR DESCRIPTION
**Pull request by c2thorn**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,15 @@
 ## 4.27.0 (Unreleased)
 
+IMPROVEMENTS:
+* clouddeploy: added `suspend` field to `google_clouddeploy_delivery_pipeline` resource ([#11914](https://github.com/hashicorp/terraform-provider-google/pull/11914))
+* compute: added maxPortsPerVm field to `google_compute_router_nat` resource ([#11933](https://github.com/hashicorp/terraform-provider-google/pull/11933))
+* compute: added `psc_connection_id` and `psc_connection_status` output fields to `google_compute_forwarding_rule` and `google_compute_global_forwarding_rule` resources ([#11892](https://github.com/hashicorp/terraform-provider-google/pull/11892))
+* containeraws: made `config.instance_type` field updatable in `google_container_aws_node_pool` ([#11892](https://github.com/hashicorp/terraform-provider-google/pull/11892))
+
+BUG FIXES:
+* compute: fixed default handling for `enableDynamicPortAllocation` to be managed by the api ([#11887](https://github.com/hashicorp/terraform-provider-google/pull/11887))
```
Comment (rileykarson) [inline CHANGELOG.md:10]:
```suggestion
* compute: fixed default handling for `enable_dynamic_port_allocation ` to be managed by the api ([#11887](https://github.com/hashicorp/terraform-provider-google/pull/11887))
```

**Review by rileykarson — APPROVED** [review]

**Comment by orlandothoeny** [comment]
Is there already a release date set for this release? Waiting for the `google_compute_forwarding_rule.psc_connection_id` attribute

**Comment by rileykarson** [comment]
Today, once I have enough time free to click the right buttons!

**Comment by rileykarson** [comment]
Done manually

**Pull request closed without merging**